cmd/cli/cmd/page.go

```diff
@@ -245,6 +245,7 @@ func KeyPageUpdate(actorUrl string, op protocol.KeyPageOperation, args []string)
 			PrintKeyUpdate()
 			return "", err
 		}
+		oldKey = newKey
```

Comment:
What does this achieve? With the change to the generate types, both fields should be optional. And the transaction executor ignores oldKey/newKey if it's not relevant to the operation.